client/components/summary/style.scss

```diff
@@ -216,6 +217,43 @@ $inner-border: $core-grey-light-500;
 	}
 }
 
+// IE11 doesn't auto-position grid children, so their position must be set manually
+@mixin set-grid-positions( $breakpoints ) {
+	@for $i from 1 through length( $breakpoints ) {
+		$number_of_items: $i;
+		$breakpoint: nth($breakpoints, $i);
+		@for $j from 1 through $number_of_items {
+			.has-#{$number_of_items}-items > .woocommerce-summary__item-container:nth-child(#{$j}) {
```

Comment:
Can we make this mixin more general (by accepting the targeted class as a parameter) since we will probably have other grids we will need to position? Or would the grid fixes in other places be different? (I'm not a CSS grid expert)

We have a mixins file this could live in if it makes sense: https://github.com/woocommerce/wc-admin/blob/master/client/stylesheets/abstracts/_mixins.scss

Reply:
> Can we make this mixin more general (by accepting the targeted class as a parameter) since we will probably have other grids we will need to position? Or would the grid fixes in other places be different? (I'm not a CSS grid expert)

Really good idea! I've just implemented it.